api/v2_recommendations.go

```diff
@@ -0,0 +1,144 @@
+//
+// Author:: Ross Moles (<ross.moles@lacework.net>)
+// Copyright:: Copyright 2022, Lacework Inc.
```

Comment:
<img src="https://media3.giphy.com/media/H3RxBUuK5rsYzs95hG/giphy.gif" width="200" />

```suggestion
// Copyright:: Copyright 2023, Lacework Inc.
```